llama.h

```diff
@@ -476,6 +476,39 @@ extern "C" {
     /// @details Accepts the sampled token into the grammar
     LLAMA_API void llama_grammar_accept_token(struct llama_context * ctx, struct llama_grammar * grammar, llama_token token);
 
+    struct llama_beam_view {
```

Comment:
//
// Beam search
//

```suggestion
    struct llama_beam_view {
```